cost/volumes/test_code/hash_dump_tester.pt

```diff
@@ -0,0 +1,122 @@
+name "Hash Dumps"
+rs_pt_ver 20180301
+short_description "Dumps hashs and the like"
+severity "medium"
+category "Cost Management"
+
+parameter "param_email" do
+  type "string"
+  label "Email address to send escalation emails to"
+end
+
+parameter "param_days_old" do
+  type "number"
+  label "Identify volumes that have been unattached for the given number of days."
+  default 30
+end
+
+resources "clouds", type: "rs_cm.clouds" do
+  filter do
+    #cloud_type ne: ["soft_layer", "cloud_stack", "rackspace_next_gen", "blue_skies","open_stack_v2","uca","open_stack_v3"]
+    cloud_type ne: ["amazon", "google", "soft_layer", "cloud_stack", "rackspace_next_gen", "blue_skies","open_stack_v2","uca","open_stack_v3"]
```

Comment:
You no longer need this.  you should remove it.

Reply:
That's just test code I didn't want to lose and I think it's helpful to keep the variations there.